test/uk/gov/hmrc/securemessage/services/SecureMessageServiceSpec.scala

```diff
@@ -55,10 +56,19 @@ class SecureMessageServiceSpec extends PlaySpec with ScalaFutures with MockitoSu
       private val result = service.createConversation(ConversationUtil.getConversationRequest(true), "cdcm", "123")
       status(result) mustBe CREATED
     }
-    "return BAD REQUEST (400) when no email address is provided" in new TestCase {
+    "return FAILED DEPENDENCY (424) when no email address is provided" in new TestCase {
```

Comment:
shall we move mocks and test data to specialised TestContext/TestCase?
see example: https://github.com/hmrc/secure-message/blob/master/test/uk/gov/hmrc/securemessage/controllers/SecureMessageControllerSpec.scala#L56